include/aws/s3/private/s3_meta_request_impl.h

```diff
@@ -226,8 +226,8 @@ struct aws_s3_meta_request {
         /* True if the finish result has been set. */
         uint32_t finish_result_set : 1;
 
-        /* To track the aws_s3_request that are active from HTTP level */
-        struct aws_linked_list ongoing_http_requests_list;
+        /* To track the aws_s3_request that are cancelled from HTTP level */
+        struct aws_linked_list cancelable_http_streams_list;
```

Comment:
I deeply apologize, but I gave a bad name suggestion. Apparently it's more common to spell this with two L's

Here is a [Cancellable](https://developer.apple.com/documentation/combine/cancellable) trait in Swift. And Apple is pretty damned choosy about names